src/init.cpp

```diff
@@ -961,25 +961,6 @@ bool AppInitParameterInteraction()
         }
     }
 
-    // Check for -debugnet
-    if (gArgs.GetBoolArg("-debugnet", false))
-        InitWarning(_("Unsupported argument -debugnet ignored, use -debug=net."));
-    // Check for -socks - as this is a privacy risk to continue, exit here
-    if (gArgs.IsArgSet("-socks"))
-        return InitError(_("Unsupported argument -socks found. Setting SOCKS version isn't possible anymore, only SOCKS5 proxies are supported."));
-    // Check for -tor - as this is a privacy risk to continue, exit here
```

Comment:
Won't this just yield a warning now?

Reply:
Not that I am aware of

Reply:
If the user has `tor=1` in bitcoin.conf, where will the InitError come from?

Reply:
Oh, I was testing `src/qt/bitcoin-qt -notor`, which gave me and still gives

![screenshot from 2018-12-20 20-07-05](https://user-images.githubusercontent.com/6399679/50305358-72be0600-04f7-11e9-9b03-352a33c0d436.png)